src/DataFixtures/Helper/FixtureHelper.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\DataFixtures\Helper;
+
+use Doctrine\Bundle\FixturesBundle\Fixture;
+use Faker\Factory as Faker;
+use Faker\Generator;
+
+abstract class FixtureHelper extends Fixture
```

Comment:
Oh :(
I'm afraid this is a no-go for me. Inheritance should be limited to actual business inheritance, shared business behaviours. Prefer composition over inheritance as much as you can.
Remove this class, and if you want you could use the `services.yaml` and the autowiring for that.